pkg/oc/admin/project/new_project.go

```diff
@@ -105,6 +108,9 @@ func (o *NewProjectOptions) complete(f *clientcmd.Factory, args []string) error
 }
 
 func (o *NewProjectOptions) Run(useNodeSelector bool) error {
+	if o.Output == nil {
```

Comment:
either move this to complete(), or compute a local var defaulting to os.Stdout... don't generally want to mutate options in the Run() method

Reply:
Of course, it turns out there are a bunch of tests that simply construct the options directly and don't run `complete()` on them (indeed, they can't since it's private). So I can either change every test, or use a local var with default like you said.